.ci/test.sh

```diff
@@ -114,8 +119,23 @@ if [[ $TASK == "swig" ]]; then
     exit 0
 fi
 
-conda install -q -y -n $CONDA_ENV cloudpickle "dask=2021.9.1" "distributed=2021.9.1" joblib matplotlib numpy pandas psutil pytest scikit-learn scipy
-pip install graphviz  # python-graphviz from Anaconda is not allowed to be installed with Python 3.9
+
+mamba install -q -y -n $CONDA_ENV \
+    cloudpickle \
+    dask \
+    distributed \
```

Comment:
maybe we could leave these just as `dask`? Since `dask` already [includes](https://github.com/conda-forge/dask-feedstock/blob/4d652a702b4a42155a89ed599e7dccbad8821f6e/recipe/meta.yaml#L21-L22) `dask-core` and `distributed`, and `dask-core` [includes](https://github.com/conda-forge/dask-core-feedstock/blob/8b195b724f280ffe795a597e8aaabd2a4a1530bd/recipe/meta.yaml#L24) `cloudpickle`